### PR TITLE
wrapper to run powershell scripts

### PR DESCRIPTION
**Pull request by ubozov**
Resolves Privatix/privatix#41

### CODE REVIEW

File: tools/dump_win/ps-runner/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Getting Started
```
Comment (furkhat) [inline tools/dump_win/ps-runner/README.md]: Why it is here? I mean in a folder structure. How is this related to dumping?

Comment (ubozov) [reply tools/dump_win/ps-runner/README.md]: This tools for run dump powershell script on Windows.
